content/en/docs/ops/configuration/traffic-management/tls-configuration/index.md

```diff
@@ -32,6 +32,7 @@ Sidecar traffic has a variety of associated connections. Let's break them down o
     By default, the sidecar will be configured to accept both mTLS and non-mTLS traffic, known as `PERMISSIVE` mode.
     The mode can alternatively be configured to `STRICT`, where traffic must be mTLS, or `DISABLE`, where traffic must be plaintext.
     The mTLS mode is configured using a [`PeerAuthentication` resource](/docs/reference/config/security/peer_authentication/).
+    In ambient, you can still create `PeerAuthentication` policy at the mesh or namespace level that uses DISABLE, but it will be ignored.
```

Comment:
After I saw https://github.com/istio/istio.io/tree/master/content/en/docs/ambient/install
It seems like PeerAuthentication not belongs to base installation in our official guide.

I think user must know this feature when they try to open this for their cluster.
In that situation, he will search mTLS related configuration so that this part will not be missed.